lib/resource-providers/certificate.ts

```diff
@@ -0,0 +1,37 @@
+import * as spi from '../spi';
+import * as acm from '@aws-cdk/aws-certificatemanager';
+import { IHostedZone } from '@aws-cdk/aws-route53';
+
+export class ImportCertificateProvider implements spi.ResourceProvider<acm.ICertificate> {
```

Comment:
While the class name is quite self-explanatory, can you please add comments to explain what this class does :)?

Reply:
Addressed